core/dbt/parser/manifest.py

```diff
@@ -1150,10 +1150,17 @@ def process_saved_queries(self, config: RuntimeConfig):
 
     def process_model_inferred_primary_keys(self):
         """Processes Model nodes to populate their `primary_key`."""
+        model_to_generic_test_map: Dict[str, List[GenericTestNode]] = {}
         for node in self.manifest.nodes.values():
             if not isinstance(node, ModelNode):
                 continue
-            generic_tests = self._get_generic_tests_for_model(node)
+            if node.created_at < self.started_at:
```

Comment:
Why we can continue here because this node is from partial parting which means it didn't change? maybe a short comment?

Reply:
Yes, because it hasn't changed on this parse. That equivalent line occurs in 19 other places, so I'm not going to bother documenting it here :)